examples/spark/spark-ml-pipe-lgb-light.py

```diff
@@ -42,49 +43,55 @@
     }
 
     with log_exec_time():
-        train_df, test_df = prepare_test_and_train(spark, path, seed)
+        train_df, test_df = prepare_test_and_train(dataset, seed)
 
-        task = SparkTask(task_type)
+        task = SparkTask(dataset.task_type)
```

Comment:
the [same](https://github.com/sb-ai-lab/SLAMA/pull/7/files#r1223141108)

Reply:
These interfaces follows interfaces of LightAutoML, which is an "origin" of SLAMA.
Not sure if it is a good idea to change these intrefaces. Need a discussion on this matter. 
(though SLAMA change them if it is necessary for performance purposes or for convenience related to Spark specific)